src/main/java/com/exactpro/th2/common/event/EventBuilder.java

```diff
@@ -50,8 +50,8 @@
 import com.google.protobuf.ByteString;
 import com.google.protobuf.Timestamp;
 
-public class Event {
-    private static final Logger LOGGER = LoggerFactory.getLogger(Event.class);
+public class EventBuilder {
```

Comment:
Please revert the class name to save backward compatibility

Reply:
Or we can inherit the Event class from EventBuilder

Reply:
I renamed it back